src/Validator/Constraints/NotEndedConferenceValidator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Validator\Constraints;
+
+use App\Entity\Conference;
+use DateTime;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class NotEndedConferenceValidator extends ConstraintValidator
+{
+    /** @var Conference */
+    public function validate($conference, Constraint $constraint): void
+    {
+        if (!$conference instanceof Conference) {
+            return;
```

Comment:
Throw exception. Look at how symfony validator works :)

Reply:
Actually I did not intend to check the data type, I just wanted the validator to return if the user did not submit anything or a blank value. I updated anyways